File: work-spring/day0904_security/src/main/java/com/grepp/day0904/model/entity/TodoEntity.java
package com.grepp.day0904.model.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

@Entity(name="todo")
public class TodoEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String username;
    private String title;
    private boolean done;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public boolean isDone() {
        return done;
    }

    public void setDone(boolean done) {
        this.done = done;
    }

    @Override
    public String toString() {
        return "TodoEntity{" +
                "id=" + id +
                ", username='" + username + '\'' +
                ", title='" + title + '\'' +
                ", done=" + done +
                '}';
    }
}

File: work-kotlin/day1021_todo/src/main/kotlin/com/grepp/todo/model/entity/TodoEntity.kt
package com.grepp.todo.model.entity

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id

@Entity(name="todo")
data class TodoEntity(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id:Int,
    var title:String,
    var done:Boolean
) {
    lateinit var username:String // 로그인 정보가 시큐리티 쪽에서 오니까 나중에 사용자이름 기록해야 되는 상황 생기더라고요.
}